charts/argo-cd/crds/crd-application.yaml

```diff
@@ -1,4 +1,4 @@
-apiVersion: apiextensions.k8s.io/v1beta1
+apiVersion: apiextensions.k8s.io/v1
```

Comment:
```suggestion
{{- if semverCompare "<1.18-0" .Capabilities.KubeVersion.GitVersion }}
apiVersion: apiextensions.k8s.io/v1beta1
{{- else }}
apiVersion: apiextensions.k8s.io/v1
{{- end }}
```

Reply:
These files are in the `crd` directory, not under `templates`. Are you sure the template directives in them will be processed?